Alpaca.Markets/RestClient.cs

```diff
@@ -26,6 +28,12 @@ public sealed partial class RestClient : IDisposable
 
         private readonly String _polygonApiKey;
 
+        private readonly IEnumerable<Int32> _retryHttpStatuses;
```

Comment:
Performance issue - you will use for fast checks and in this case HashSet<Int32> or SortedSet<Int32> looks more appropriate container type.

Reply:
Good point.  Switched to HashSet.